sms.info.yml

```diff
@@ -8,3 +8,5 @@ configure: admin/config/smsframework
 dependencies:
   - telephone
   - dynamic_entity_reference
+test_dependencies:
```

Comment:
Ok. As an experiment, I have committed / pushed a change with the test_dependencies removed from sms.info.yml. Let's see what happens after the dependencies are rebuilt.